indico/modules/events/surveys/templates/management/survey.html

```diff
@@ -209,4 +219,4 @@ <h2>
                data-submission-id="{{ submission.id }}"></a>
         </td>
     </tr>
-{% endmacro %}
+{% endmacro %}
```

Comment:
Please configure your editor to end each file with a linebreak. In VSCode that's `"files.insertFinalNewline": true``